src/config/cacheConfig.ts

```diff
@@ -0,0 +1,17 @@
+import { ConfigService } from '@nestjs/config';
+import { redisStore } from 'cache-manager-redis-yet';
+
+export async function buildCacheConfig(
+  configService: ConfigService,
+): Promise<any> {
+  return {
+    store: await redisStore({
+      socket: {
+        host: configService.get('redisHost'),
+        port: configService.get('redisPort'),
+      },
+      password: configService.get('redisPassword'),
+      database: 2,
```

Comment:
@rattrap in case this is possible to be specified with the GCP Redis I'd keep this for monitoring reasons